article/sections/_2_graph_importance.tex

```diff
@@ -0,0 +1,189 @@
+   \caption{Causal Graph for the Walk-Transit-Walk Utility Function}
+   \label{fig:WTW_causal_2}
+\end{figure}
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{WTD_interacting_graph.png}
+   \caption{Causal Graph for the Walk-Transit-Drive Utility Function}
+   \label{fig:WTD_causal_2}
+\end{figure}
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{DTW_interacting_graph.png}
+   \caption{Causal Graph for the Drive-Transit-Walk Utility Function}
+   \label{fig:DTW_causal_2}
+\end{figure}
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{WALK_interacting_graph.png}
+   \caption{Causal Graph for the Shared Ride 3+ Utility Function}
+   \label{fig:WALK_causal_2}
+\end{figure}
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{BIKE_interacting_graph.png}
+   \caption{Causal Graph for the Bike Utility Function}
+   \label{fig:BIKE_causal_2}
+\end{figure}
+
+Using the setup described above, we conduct our simulation exercise as follows:
+\begin{enumerate}
+
+\item We build a DAG (Figure \ref{fig:IND_GRAPH}) where we assume all explanatory variables are independent of each other. 
+\item We simulate data based on this graph and generate outcomes based on a chosen ``true'' multinomial logit model.
+\item We predict outcomes based on the mode choice model.
+\item We apply the do-operator \citet{pearl_causality_2000} to our simulated data to reduce the travel distance for all individuals in the dataset. This emulates a company's decision to move its employees closer to campus.
+\item We predict new outcomes based on the mode choice model.\newline
+\item We build a different DAG (Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2}) for each utility function based on assumptions of how explanatory variables interact and influence the outcome.
+\item We use the real dataset to estimate the relationships between different variables outlined in Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2}.
+\item We simulate data for variables without parent nodes in Figure Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2} and we use the estimated relationships from step 6 to simulate the remaining explanatory variables.
+\item We then use the choice model to simulate outcome choices based on data simulated for Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2}.
+\item We apply the do-operator, as above, to reduce the travel distance for all individuals in the dataset and to emulate a company's decision to move its employees closer to campus.
+\item We use the estimated relationships for this causal graph to simulate all the explanatory variables in Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2}(namely the variables affected by travel distance).
+\item We use the estimated choice model to produce outcomes based on the simulated data from the previous step.
```

Comment:
I think those bullets, the way they're written now, are more confusing than helpful. They're more in the weeds of how the simulation was done mechanically, but they don't convey the bigger picture of what we're trying to accomplish. I suggest replacing with a higher level bullet list that conveys the idea we're trying to illustrate, as opposed to the mechanics. Let's discuss tomorrow.